src/composables/queries/usePoolsQuery.ts

```diff
@@ -104,6 +101,18 @@ export default function usePoolsQuery(
     };
   }
 
+  function buildProviders() {
+    const providers: SDKPoolRepository[] = [];
+    if (isBalancerApiDefined()) {
+      const balancerApiRepository = initializeDecoratedAPIRepository();
+      providers.push(balancerApiRepository);
+    }
+    const subgraphRepository = initializeDecoratedSubgraphRepository();
+    providers.push(subgraphRepository);
+
+    return providers;
+  }
```

Comment:
Lets stick to the naming convention already established.
```suggestion
  function buildRepositories() {
    const repositories: SDKPoolRepository[] = [];
    if (isBalancerApiDefined()) {
      const balancerApiRepository = initializeDecoratedAPIRepository();
      repositories.push(balancerApiRepository);
    }
    const subgraphRepository = initializeDecoratedSubgraphRepository();
    repositories.push(subgraphRepository);

    return repositories;
  }
```